JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.ts

```diff
@@ -0,0 +1,278 @@
+
+  fontStyleOptions: any;
+
+  listOptions: any;
+
+  dateBoxOptions: any;
+
+  checkBoxOptions: any;
+
+  attachOptions: any;
+
+  addButtonOptions: any;
+
+  removeButtonOptions: any;
+
+  aboutButtonOptions: any;
+
+  textAlignOptions: any;
+
+  constructor(service: Service) {
+    this.multiline = true;
+    this.colors = service.getColors();
+    this.fontSizes = service.getFontSizes();
+    this.lineHeights = service.getLineHeights();
+    this.lineHeight = [this.lineHeights[1].lineHeight];
+    this.fonts = service.getFonts();
+    this.fontStyles = service.getFontStyles();
+    this.textAligns = service.getTextAligns();
+    this.selectedTextAlign = [this.textAligns[0].alignment];
+    this.listTypes = service.getListType();
+
+    this.undoButtonOptions = {
+      icon: 'undo',
+      onClick: () => {
+        notify('Undo button has been clicked!');
+      },
+    };
+
+    this.redoButtonOptions = {
+      icon: 'redo',
+      onClick: () => {
+        notify('Redo button has been clicked!');
+      },
+    };
+
+    this.colorPickerOptions = {
+      items: this.colors,
+      icon: 'square',
+      dropDownOptions: { width: 'auto' },
+      dropDownContentTemplate: 'colorpicker',
+      onInitialized: ({ component }): void => {
+        this.colorPicker = component;
+      },
+    };
+
+    this.fontSizeOptions = {
+      displayExpr: 'text',
+      keyExpr: 'size',
+      useSelectMode: true,
+      items: this.fontSizes,
+      selectedItemKey: this.fontSizes[2].size,
+      itemTemplate: 'fontSizeTemplate',
+    };
+
+    this.lineHeightOptions = {
+      icon: 'indent',
+      displayExpr: 'text',
+      keyExpr: 'lineHeight',
+      useSelectMode: true,
+      items: this.lineHeights,
+      selectedItemKey: this.lineHeight,
+      onSelectionChanged: (e: { item: { lineHeight: number } }): void => {
+        this.lineHeight = [e.item.lineHeight];
+      },
+    };
+
+    this.fontSelectorOptions = {
+      placeholder: 'Font',
+      displayExpr: 'text',
+      dataSource: new DataSource(this.fonts),
+    };
+
+    this.fontStyleOptions = {
+      displayExpr: 'text',
+      items: this.fontStyles,
+      keyExpr: 'style',
+      stylingMode: 'outlined',
+      selectionMode: 'multiple',
+      onItemClick: (e: { itemData: { hint: string } }): void => {
+        notify(`The "${e.itemData.hint}" button was clicked`);
+      },
+    };
+
+    this.listOptions = {
+      items: this.listTypes,
+      keyExpr: 'alignment',
+      stylingMode: 'outlined',
+      onItemClick: (e: { itemData: { hint: string } }): void => {
+        notify(`The "${e.itemData.hint}" button was clicked`);
+      },
+    };
+
+    this.dateBoxOptions = {
+      width: 200,
+      type: 'date',
+      value: new Date(2022, 9, 7),
+    };
+
+    this.checkBoxOptions = {
+      value: false,
+      text: 'Private',
+      onOptionChanged: (): void => {
+        notify('Private option has been changed!');
+      },
+    };
+
+    this.attachOptions = {
```

Comment:
I think it's better to name it consistently with another buttons' names. Let's change it everywhere.
```suggestion
    this.attachButtonOptions = {
```

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/5ba43954aa5e53c1a0c2a8596281ef2bf941d78c